project.py

```diff
@@ -92,10 +99,10 @@ def get_mask(self, image):
         flags |= cv.FLOODFILL_MASK_ONLY
         flags |= 255 << 8
         seed = (100, 100)
-        cv.floodFill(image, mask, seed, (255,255,255), (self.lo,)*3,
-                        (self.hi,)*3, flags)
-        kernel = np.ones((5,5), np.uint8)
-        mask = cv.dilate(mask, kernel, iterations = 3)
+        cv.floodFill(image, mask, seed, (255, 255, 255), (self.lo,)*3,
+                     (self.hi,)*3, flags)
+        kernel = np.ones((1, 1), np.uint8)
```

Comment:
@Foxiy This line is actually crucial. I didn't put it explicitly into the commit message because I'd like to know what you think about it.